### DISCUSSION TITLE
Help - ystem Log Shows Audio capture but ..

### DISCUSSION THREAD

**Discussion by pjdreger3** [mcguirepr89/BirdNET-Pi#589] [Q&A]
The System log shows a stream of captures that show up in the BirdSong directory but nothing ever shows up in the Processed directory. 

And looking at the system log it's always processing the same sound - a Emerald Chinned Hummingbird which isn't found in Georgia. And I see no detections at all. 
Any pointers greatly appreciated

**Comment by ehpersonal38** [comment]
HI @pjdreger3 ! 

This is a tell-tale sign of BirdNET-Pi not receiving any data from your microphone.

The following guide is a good starting point for verifying that the microphone is detected by your system, and that the proper gain (volume) levels are set: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card

**Reply by pjdreger3** [reply]
did that and maxed usb sound card input . rebooted. the long now shows a continuous capturing of a Northern Bobwhite but no detections.

**Reply by ehpersonal38** [reply]
When you listen to the Spectrogram what do you hear?

**Reply by pjdreger3** [reply]
I hear nothing connected via the headphone jack on the Pi or via the soundcard

**Reply by ehpersonal38** [reply]
Okay well your microphone is not working, whether that's a hardware or software issue I'm not sure. I'd try the mic on a computer (Windows, Mac or Linux) and see if it works there
